### PR TITLE
fix(docs): note that 2.14 doesn't work with Jupyter

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

Jupyter notebook relies on tools, especially `opentrons.simulate.get_protocol_api()`, that aren't supported in version 2.14 and later of the Python API. We noted this on the Versioning page but didn't mention it on the Advanced Control page. This PR updates the Advanced Control page.

# Test and Deployment Plan

This is a _docs hotfix_ and requires a slightly different process than most PRs. `edge` has diverged significantly from the previous docs deployment, and we don't want to include updates from edge in this small docs change.

1. Get approving reviews based on the content in the [branch sandbox](http://sandbox.docs.opentrons.com/docs-hotfix-jupyter-api-version/v2/new_advanced_running.html).
  a. Approve the additions to Advanced Control.
  b. Check that the API Reference and other pages don't have any extra new stuff in them.
2. I'll tag and deploy the docs _before_ merging this PR.
3. I'll merge this PR so the changes carry forward in future docs.

# Changelog

- New note about Jupyter version support.
- Changed version in code snippet from variable (latest version) to static `2.13`

# Review requests

- Check out the sandbox. Please double check the API reference and other snippets that have variable API levels! They should all reflect 2.14 as of its release.

# Risk assessment

V low, it's docs.

### CODE REVIEW

File: api/docs/v2/new_advanced_running.rst [hunk 1]
```diff
@@ -32,7 +35,7 @@ Rather than writing a  ``run`` function and embedding commands within it, start
     :substitutions:
 
     import opentrons.execute
-    protocol = opentrons.execute.get_protocol_api('|apiLevel|')
+    protocol = opentrons.execute.get_protocol_api('2.13')
```
Comment (SyntaxColoring) [inline api/docs/v2/new_advanced_running.rst:38]: I think you'll want to make the same change to the example down the page, on line 63.

**Review by SyntaxColoring — COMMENTED** [review]
One suggestion for the content.

**Review by SyntaxColoring — COMMENTED** [review]
And some suggestions for the deploy process. If you're already following a plan you worked out with @shlokamin, feel free to ignore this!

-----

If I understand your deployment plan correctly, you're intending to do this:

```
old edge (docs@2.14_1)
|     \
...    \
|      "fix version + note" (this PR's branch)
...    last commit will receive new `docs@` tag, and be deployed
|      |
...    |
|      |
|      /
|     /
edge
```

This branch looks correctly based on `docs@2.14_1` (so, it doesn't include any changes to to the docs, except for this hotfix). 👍 

The tag+merge plan makes sense to me, with one suggestion. When you do the merge into `edge`, I would select a normal merge from the GitHub dropdown, *not* the default squash merge.

The upside to a normal merge is that Git will understand that the literal, exact commit that you tagged for deployment is part of `edge`'s history. This avoids confusing conflicts in case we need to do any more hotfixes atop this one.

The downside is that you won't have a chance to revise commit messages, so `edge` will accidentally have a commit message like `fix version + note`, which doesn't follow our usual convention. There are ways to fix this, but I wouldn't bother. This happens occasionally and it's no big deal.

**Review by jwwojak — COMMENTED** [review]
In the intro sentence to the bullet list, the "of the Python API" text reads as wee bit awkward to me. 

**Suggestion:** identify "Python API" earlier in the sentence and get rid of the of:

> The Jupyter Notebook server only supports Python API versions 2.13 and earlier.

With a sentence-fragment bullet list, try starting each bullet with a verb -ing:

> You can access the OT-2's Jupyter Notebook by either:

> - Clicking the **Advanced** tab of Robot Settings and then clicking **Launch Jupyter Notebook**.
> - Typing your robot's IP address in the address field of your browser (e.g. ``http://<robot-ip>:48888``). 

Thanks!

File: api/docs/v2/new_advanced_running.rst (hunk 1 above)
Comment (ecormany) [reply api/docs/v2/new_advanced_running.rst:38]: good catch, ty.

**Comment by ecormany** [comment]
> And some suggestions for the deploy process. If you're already following a plan you worked out with @shlokamin, feel free to ignore this!

I almost put "do not show me ascii art branch diagrams, they make my eyes glaze over" in the description 😝 

I see the benefit of the regular merge. In the end, this PR is going to have 3 or 4 total commits. Is that a problem? I could "manually squash" all the changes together, close this PR, and open a new PR with a single commit if that would make everyone's lives better. @SyntaxColoring @shlokamin

**Comment by SyntaxColoring** [comment]
> I see the benefit of the regular merge. In the end, this PR is going to have 3 or 4 total commits. Is that a problem? I could "manually squash" all the changes together, close this PR, and open a new PR with a single commit if that would make everyone's lives better. @SyntaxColoring @shlokamin

Yeah, I'd say that's worth a shot. Thanks.

**Comment by ecormany** [comment]
Closing in favor of a forthcoming single-commit PR.

**Pull request closed without merging**